MdsSupportingClasses/Collivery.php

```diff
@@ -543,28 +546,31 @@ public function getAddress($address_id)
      * @return array
      * @throws SoapConnectionException
      */
-    public function getAddresses(array $filter = [])
+    public function getAddresses($filter = [])
     {
         if (($this->check_cache == 2) && empty($filter) && $this->cache->has('collivery.addresses.'.$this->client_id)) {
             return $this->cache->get('collivery.addresses.'.$this->client_id);
         } else {
```

Comment:
There's no need for this `else {}` as it `return`s inside of the `if {}`

Reply:
It'll only return a value if the addresses have been cached. If they've not been cached it will return nothing.
Unless I misunderstood your message.

Reply:
Don't worry